client/src/containers/Gallery/Gallery.js

```diff
@@ -944,6 +943,8 @@ class Gallery extends Component {
       cssClasses.push('gallery__main--has-opened-item');
     }
 
+    const isEnabled = this.props.view === 'tile' && this.props.type === ACTION_TYPES.ADMIN;
```

Comment:
```suggestion
    const canSelect = this.props.view === 'tile' && this.props.type === ACTION_TYPES.ADMIN;
```
`isEnabled` out here looks like it's talking about the enabled state of the gallery

Reply:
Updated